pkg/monitor/backenddisruption/disruption_backend_sampler_test.go

```diff
@@ -84,7 +84,8 @@ func TestBackendSampler_checkConnection(t *testing.T) {
 				connectionType:        NewConnectionType,
 				path:                  "/302-bad-response",
 			},
-			wantErr: true,
+			// TODO: should this be error when Location header is not set?
+			wantErr: false,
```

Comment:
This is passing on other PRs I think, any idea what's in here that's causing this? Not super familiar with the sampler code yet.

Reply:
Looks like this is coming from go 1.18 -> 1.19 change, apparently missing `Location` header is acceptable.

Reply:
Yup, found it here: https://github.com/golang/go/commit/a41763539c7ad09a22720a517a28e6018ca4db0f